Dockerfile

```diff
@@ -31,17 +32,19 @@ RUN go mod download
 # Copy the go source
 COPY main.go main.go
 COPY api/ api/
+COPY certs/ certs/
+COPY cloudinit/ cloudinit/
```

Comment:
Why not just copy '.' instead?

Reply:
+1 to `COPY ./ ./`

Reply:
I followed the kubebuilder template, but no strong preferences, so updated.